docs/2_database.rst

```diff
@@ -1,9 +1,11 @@
-Connecting to the database
+Database Configuration
 ==========================
```

Comment:
the header markup need to be the same number of characters as the text above it (rst standard)